operations/helm/charts/mimir-distributed/values.yaml

```
@@ -2136,6 +2136,9 @@ nginx:
           location = /api/v1/push {
             proxy_pass      http://{{ template "mimir.fullname" . }}-distributor-headless.{{ .Release.Namespace }}.svc.{{ .Values.global.clusterDomain }}:{{ include "mimir.serverHttpListenPort" . }}$request_uri;
           }
+          location /otlp {
```

[nit] Should we be more explicit and specify the actual endpoint? Just in case in the future we'll support some read path endpoints too, which will route to other components instead.

```suggestion
          location /otlp/v1/metrics {
```